flamegraph.pl

```diff
@@ -555,6 +555,37 @@ sub read_palette {
 my %Node;	# Hash of merged frame data
 my %Tmp;
 
+# Regexp for parsing out stack data from sample count(s)
+#
+# Effectively, we parse out the 1 (or 2) sample sizes from the end of the
+# line, and assume everything else (PREMATCH) is the stack data.
+my $line_regexp = qr/
+  \s+                # Space after stack
+  (\d+(?:\.\d*)?)    # First sample size
```

Comment:
Wasn't sure if the decimal would ever be valid in a sample count, but left it just in case for backwards compatibility.  Was considering removing it, but decided against it.